lib/bot.js

```diff
@@ -36,6 +36,8 @@ class Bot {
     this.channels = _.values(options.channelMapping);
     this.ircStatusNotices = options.ircStatusNotices;
     this.announceSelfJoin = options.announceSelfJoin;
+    this.ignoreBots = options.ignoreBots;
+    this.ignoreIrcNicks = options.ignoreIrcNicks || [];
```

Comment:
Oh, yeah, it could be a better idea since it doesn't change. It's less readily accessible, though, which might make `username#discriminator` better, especially since usernames and discriminators don't, I think, change much? (I'd go about getting the ID by either putting Discord in dev mode or using `\@Username` to grab the ID from the escaped mention, both of which seem more awkward than `username#discriminator`.)